src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -25,6 +25,7 @@
 #include "precompiled.hpp"
 #include "classfile/classLoaderDataGraph.hpp"
 #include "classfile/javaClasses.inline.hpp"
+#include "classfile/vmSymbols.hpp"
```

Comment:
I assume this is a transitive include, so good to put it in the file where it's used.